package/plugin-gargoyle-ddns/files/www/js/ddns.js

```diff
@@ -535,6 +542,20 @@ function setDocumentFromUci(srcUci, pkg, section)
 	var forceDays = (getMultipleFromUnit( srcUci.get("ddns_gargoyle", section, "force_unit") ) * srcUci.get("ddns_gargoyle", section, "force_interval"))/(24*60*60);
 	forceDays = (forceDays > 0) ? forceDays : 3;
 	document.getElementById( "ddns_force" ).value = forceDays;
+	
+	var ip_from=srcUci.get("ddns_gargoyle",section,"ip_interface");//2020-03-21
+	var selectip = document.getElementById("ip_from");
+	if(ip_from){
```

Comment:
Please replace this section with setSelectedValue or srtSelectedText (whichever is most appropriate)